ci/aws/byol.pkr.hcl

```diff
@@ -107,7 +99,7 @@ build {
     source      = "utils/10-run-tyk.conf"
   }
   provisioner "shell" {
-    environment_vars = ["VERSION=${var.version}" , "GEOIP_LICENSE=${var.geoip_license}"]
+    environment_vars = ["VERSION=${var.version}" , "GEOIP_LICENSE=${var.geoip_license}" ]
```

Comment:
This line has a whitespace issue. In other places (release.yml) the whitespace is being cleaned up. Might be worth to give a once over for the standard ` , `, `[ `, ` ]` (is there a `fmt` tool for this?)

Reply:
This is an issue in the parent template itself, will fix it there. (https://github.com/TykTechnologies/gromit/blob/c1cce7f97442d1ae5891cb5c343cdf64f67dca0e/policy/templates/releng/ci/aws/byol.pkr.hcl#L113)